symengine/flint_wrapper.h

```diff
@@ -627,6 +667,13 @@ class fmpz_poly_wrapper
         return fmpz_poly_divrem(*q.get_fmpz_poly_t(), *r.get_fmpz_poly_t(),
                                 *get_fmpz_poly_t(), *b.get_fmpz_poly_t());
     }
+
+    fmpz_poly_factor_wrapper factors() const
+    {
+        fmpz_poly_factor_wrapper r;
+        fmpz_poly_factor(r.get_fmpz_poly_factor_t(), poly);
```

Comment:
For version <= 2.5.2, you can throw a std::runtime_error and then check for that in the tests